docs/resources/Channel.md

```diff
@@ -700,6 +705,8 @@ This endpoint supports requests with `Content-Type`s of both `application/json`
 > info
 > Note that when sending `application/json` you must send at least one of `content` or `embed`, and when sending `multipart/form-data`, you must send at least one of `content`, `embed` or `file`. For a `file` attachment, the `Content-Disposition` subpart header MUST contain a `filename` parameter.
 
+You may create a message as a reply to another message. To do so, include a [`message_reference`](#DOCS_RESOURCES_CHANNEL/message-object-message-reference-structure) with a `channel_id` and `message_id`. This requires the `VIEW MESSAGE HISTORY` permission, the reference message cannot be a system message, and the channel id of the referenced message must match the channel that you're sending the message in.
```

Comment:
looks like the client also sends `guild_id`, does it matter whether that's sent?

also, which IDs are present in the message_reference object for replies? different message types contain different sets of ids

Reply:
Wait, so they send guild_id on REST responses for the inline replies, but not on the messages themselves?

Or am I misunderstanding this

Reply:
when you try to reply to a message via the client, it sends the message with all 3 `message_reference` fields

Reply:
Now only a `message_id` is required when sent